installer/common/shared.sh

```diff
@@ -225,7 +225,13 @@ helm upgrade $EFS_SERVER_NAME stable/$EFS_CHART_NAME \
 --install \
 --namespace "$1" \
 --timeout 600 \
+--set efsProvisioner.awsRegion="$3" \
 --set efsProvisioner.dnsName="$2.efs.$3.amazonaws.com" \
 --set efsProvisioner.efsFileSystemId="$2" \
---set efsProvisioner.awsRegion="$3"
+--set efsProvisioner.path="/" \
+--set efsProvisioner.provisionerName="kubernetes.io/aws-efs" \
+--set efsProvisioner.storageClass.name="pv-volume" \
```

Comment:
You need to update the sensor-data-scala app yaml file if you change this now this is broken.
Why not leave the default here which is aws-efs?